source/elements/oneDAL/source/introduction.rst

```diff
@@ -1,3 +1,20 @@
 ============
 Introduction
 ============
+
+oneDAL is library that provides building blocks covering all stages of data analytics: data acquisition
+ from a data source, preprocessing, transformation, data mining, modeling, validation, and decision making.
+
+Library consists of the following major components: Data Management, Algorithms, and Services
+
+The Data Management component includes classes and utilities for data acquisition, initial preprocessing
+ and normalization, for data conversion into numeric formats done by one of supported Data Sources,
+ and for model representation.
+
+The Algorithms component consists of classes that implement algorithms for data analysis (data mining),
+ and data modeling (training and prediction). These algorithms include matrix decompositions, clustering,
```

Comment:
I'd update the examples of the algorithms to highlight the most important we are aware of

Reply:
not fixed

Reply:
You mean particular algorithm names (like PCA, KNN, etc) for each category ?